integrations/scalyr/event.go

```diff
@@ -11,7 +11,18 @@ func (e *Event) MarshalJSON() ([]byte, error) {
 	data := make(map[string]interface{})
 	data["ts"] = e.Timestamp
 	data["sev"] = e.Severity
-	data["attrs"] = e.Data
+
+	attributes := e.Data
+
+	message, err := json.Marshal(e.Message)
+
+	if err != nil {
+		return nil, err
```

Comment:
What happens when there's an error? Are we logging it?

Reply:
We're sending the error back to stdout: https://github.com/ConvertKit/stories/blob/master/main.go#L79

Right now, I don't want to add too much to it in terms of fallback/notification as I don't know what exactly we want it to be. Maybe we could talk about it as we use it more?